package excecao;

public class Basico {
	
	public static void main(String[] args) {
		
		Aluno a1 = null;
		
		try {
			imprimirAluno(a1);
		} catch(Exception excecao) {
			System.out.println("Ocorreu um erro para imprimir o nome do usu√°rio");
		}
		
		try {
			System.out.println(7 / 0);
		} catch (ArithmeticException e) {
			System.out.println("Ocorreu um erro: " + e.getMessage() );
		}
	}
	
	public static void imprimirAluno(Aluno aluno) {
		System.out.println(aluno.nome);	
	}
}
